actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -73,6 +70,9 @@ def flush_newline_if_pending(src)
         end
       end
 
+      # For backwards compatibility
+      Erubis = Erubi
```

Comment:
I think the body of the Erubis class (and now the Erubi class) should have been / should be fully `:nodoc:` -- its existence should be documented, but not any of its methods.

As it was published, though, it's probably safer to keep the old one as is (but deprecate it). 

Let's throw a `:nodoc: all` inside `Erubi`, though.